examples/next/pages/ui/components/storage/storage-browser/default-auth/aws-exports.js

```diff
@@ -1,2 +1,2 @@
-import awsExports from '@environments/storage/file-uploader/src/aws-exports';
+import awsExports from '@environments/storage/gen2/amplify_outputs.json';
```

Comment:
updated this since gen2 outputs will be the only thing that will work for default auth